### DISCUSSION TITLE
transaction hash labels (just cow protocol for now)

### DISCUSSION THREAD

**Discussion by andrewhong5297** [duneanalytics/spellbook#2810] [Sector: Labels]
We're looking to consolidate all kinds of labels (i.e. models that use the `label` tag) under the `labels` folder. Using directories instead of tags to organize will make navigation and maintenance easier long term.

This means moving the [tx hash labels in cowswap](https://github.com/duneanalytics/spellbook/tree/main/models/cow_protocol/tx_hash_labels) in as well, however these should not be appended to the address labels table. 

My only feedback is that `category` should likely be `protocol`, the `label_type` should be `event_type` instead (like “swap, hack, payment, governance, etc”), and the current `category` should be moved to `model_name`. 

Thoughts @gentrexha @0xRobin @jeff-dude ?

**Comment by jeff-dude** [comment]
fyi @Hosuke @augustog @bh2smith

**Reply by bh2smith** [reply]
I think this is more relevant to @gentrexha  who has been the most active with these labels. I'll leave it with him. 

Otherwise sounds fine by me.

**Reply by busofu3s0zou** [reply]
thats so easy

**Comment by gentrexha** [comment]
That's fine by me. I would appreciate it if we could merge all my current open PRs and move them together instead of having to change every single PR now.

**Reply by andrewhong5297** [reply]
Yup, will do!

**Reply by gentrexha** [reply]
Thanks!

**Comment by andrewhong5297** [comment]
fyi we are moving things around so that `labels.all` is now `labels.addresses`, and `labels.transactions` will be its own new section.

https://github.com/duneanalytics/spellbook/pull/3040
